src/__tests__/calculate.test.js

```diff
@@ -0,0 +1,62 @@
+import calculate from '../logic/calculate';
+
+describe('Testing the calculator operations and sum', () => {
+  test('+/- button should make a positive number negative', () => {
+    const result = calculate({ total: '20', next: '', operation: '' }, '+/-');
+    const { total } = result;
+    expect(total).toEqual('-20');
+  });
+
+  test('+/- button should make a negative number positive', () => {
+    const result = calculate({ total: '-20', next: '', operation: '' }, '+/-');
+    const { total } = result;
+    expect(total).toEqual('20');
+  });
+
+  test('% button should return its percentage', () => {
+    const result = calculate({ total: '50', next: '', operation: '' }, '%');
+    const { total } = result;
+    expect(total).toEqual('0.5');
+  });
+
+  test('+ button should add 2 numbers and return its correct total', () => {
+    const result = calculate({ total: '50', next: '20', operation: '+' }, '=');
+    const { total } = result;
+    expect(total).toEqual('70');
+  });
+
+  test('- button should subtract a number and return its correct total', () => {
+    const result = calculate({ total: '50', next: '30', operation: '-' }, '=');
+    const { total } = result;
+    expect(total).toEqual('20');
+  });
+
+  test('x button should multiply two numbers and return its correct total', () => {
+    const result = calculate({ total: '10', next: '20', operation: 'x' }, '=');
+    const { total } = result;
+    expect(total).toEqual('200');
+  });
+
+  test('÷ button should divide a number and return its correct sum', () => {
+    const result = calculate({ total: '100', next: '20', operation: '÷' }, '=');
+    const { total } = result;
+    expect(total).toEqual('5');
+  });
```

Comment:
- [ ] [REQUIRED] I got an error when I divide a number by zero. It will be nice if you fix it and add test unit for that case.
![1](https://user-images.githubusercontent.com/58552748/139542888-8622f890-013c-4b2f-a7b1-14e8c23ef9c3.jpg)